packages/shared/lib/auxiliary/icon/constants/account-actions-icons.ts

```diff
@@ -0,0 +1,10 @@
+import { Icon } from '../enums'
+
+export const ACCOUNT_ACTIONS_ICONS = {
```

Comment:
Not sure if we need to define these here as they are only used in one place. But perhaps you like the pattern of semantically naming them and future proofing the use if they are used in multiple places like the settings icon. I will let you decide.

Reply:
updated